packages/web-ui/package.json

```diff
@@ -80,6 +80,7 @@
     "tailwind-merge": "^2.4.0",
     "tailwindcss": "^3.4.4",
     "tailwindcss-animate": "^1.0.7",
+    "typescript": "^5.8.2",
```

Comment:
Do we need to put a typescript in this package? Isn't it inherited by our typescript package?

Reply:
nope se me ha escapao